src/AttrDef/HTML/LinkTypes.hack

```diff
@@ -67,13 +67,6 @@ class HTMLPurifier_AttrDef_HTML_LinkTypes extends HTMLPurifier\HTMLPurifier_Attr
             $ret_lookup[$part] = true;
         }
 
-        // make all allowed exist
```

Comment:
Can you give a tldr on why removing this helps and what it was originally supposed to do?

Reply:
In the original implementation of LinkType: https://github.com/ezyang/htmlpurifier/blob/b81690c17eae5dbea7557d1f359af8dd17e9beef/library/HTMLPurifier/AttrDef/HTML/LinkTypes.php#L44-L70
It doesn't have this check, not sure how this check gets introduced.

Reply:
It was an attempt to fix a diff: https://slack-github.com/slack/htmlsanitizer-hack/pull/94

🙃

Reply:
This is definitely what adds `noopener noreferer` to the rel regardless of what is already in the rel, if there even is a rel or a target etc.

Reply:
This effectively changed the config var `Attr.AllowedRel` to really mean `Attr.RequiredRel` or `Attr.DefaultRel`